doctest for 0-add_integer.py
============================
This function add two add_integer

>>> add_integer = __import__('0-add_integer').add_integer

Test for success
================
>>> add_integer(1,2)
3

>>> add_integer (50, -8)
42

>>> add_integer(-2, 20)
18

>>> add_integer(1.5, 1.5)
2

>>> add_integer(100.3, -2)
98

>>> add_integer(20, 0.2)
20

>>> add_integer(20)
118

==================
Test for fail
==================

>>> add_integer(2, "baguette")
Traceback (most recent call last):
TypeError: b must be an integer

>>> add_integer("fromage", 2)
Traceback (most recent call last):
TypeError: a must be an integer

>>> add_integer("fromage", "baguette")
Traceback (most recent call last):
TypeError: a must be an integer

>>> add_integer()
Traceback (most recent call last):
TypeError: add_integer() missing 1 required positional argument: 'a'

>>> add_integer(None)
Traceback (most recent call last):
TypeError: a must be an integer

>>> add_integer(1, 2, 3)
Traceback (most recent call last):
TypeError: add_integer() takes from 1 to 2 positional arguments but 3 were given

>>> add_integer(1, None)
Traceback (most recent call last):
TypeError: b must be an integer